appcues/src/main/java/com/appcues/statemachine/Action.kt

```diff
@@ -6,7 +6,11 @@ internal sealed class Action {
     data class StartExperience(val experience: Experience) : Action()
     data class StartStep(val stepReference: StepReference) : Action()
     object RenderStep : Action()
-    data class EndExperience(val markComplete: Boolean, val destroyed: Boolean) : Action()
+    data class EndExperience(
+        val markComplete: Boolean,
+        val destroyed: Boolean,
+        val trackAnalytics: Boolean = true,
```

Comment:
this flag can now be used to optionally disable analytics when closing out an experience - for cases like this reset or the existing `stop()` feature. Open to any better ideas, but this was first approach I could think of.